L1Trigger/L1TTwinMux/plugins/L1TTwinMuxProducer.cc

```diff
@@ -0,0 +1,101 @@
+//-------------------------------------------------
+//
+//   Class: L1TwinMuxProducer
+//
+//   L1TwinMuxProducer EDProducer
+//
+//
+//   Author :
+//   G. Flouris               U Ioannina    Feb. 2015
+//   Mod.: g Karathanasis 
+//--------------------------------------------------
+
+
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include <FWCore/Framework/interface/ConsumesCollector.h>
+#include <FWCore/Framework/interface/one/EDProducer.h>
```

Comment:
If the L1TTwinMuxProducer inherits from edm::EDProducer this is the wrong include file.